integration/up/autocreate_test.go

```diff
@@ -192,7 +204,14 @@ func TestUpAutocreateV2(t *testing.T) {
 	require.NoError(t, commands.RunOktetoCreateNamespace(oktetoPath, namespaceOpts))
 	defer commands.RunOktetoDeleteNamespace(oktetoPath, namespaceOpts)
 	require.NoError(t, commands.RunOktetoKubeconfig(oktetoPath, dir))
-	c, _, err := okteto.NewK8sClientProvider().Provide(kubeconfig.Get([]string{filepath.Join(dir, ".kube", "config")}))
+	config := kubeconfig.Get([]string{filepath.Join(dir, ".kube", "config")})
+	for _, authInfo := range config.AuthInfos {
+		homeEnvVar := api.ExecEnvVar{Name: constants.OktetoHomeEnvVar, Value: dir}
+		authInfo.Exec.Env = append(authInfo.Exec.Env, homeEnvVar)
+	}
+
+	c, _, err := okteto.NewK8sClientProvider().Provide(config)
```

Comment:
2 things to mention here:
a) I couldn't find a simpler way to do this. Please comment if you think of some
b) If we want to go ahead and add this to all tests, we should probably refactor to simplify writing tests and add this kind of logic to the `kubeconfig.Get func`